rvepak/mailroom.py

```diff
@@ -0,0 +1,137 @@
+import os
+import sys
+import prettytable
+
+donors = [('Jack', 1), ('John', 2), ('Ray', 3), ('Kelly', 2), ('Rick', 1)]
+
+maxTriesCount = 0
+
+
+def inputPrompts():
+    '''
+    this method prompts for the 3 values
+    '''
+    global maxTriesCount
+    allowedResponsesList = allowedResponses(allowedResponsesMap)
```

Comment:
Camel casing is for Java and C.  The Pythonistas use underscores for all their variable names.
`allowed_responses_list = allowed_responses(allowed_responses_map)`

We will get into PEP8 and the conventions used in python a bit later I think.